ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ingress/lbtype/nlb/subnets/provision/cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb-subnets-provision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb-subnets-provision
+  steps:
+  - chain: ipi-conf-aws
+  - ref: ingress-aws-conf-lbtype-nlb
+  - chain: ipi-install
+  - ref: ingress-enable-http2-globally
+  - ref: ingress-aws-conf-subnets
```

Comment:
So this is a day2 operation, right? If yes, suggest to rename the install workflow to including `day2` keyword to indicated that.

Reply:
Actually this is temporary workaround to enable subnets integration test and test it early, once the installer epic is completed we will move the step to day1 with updating subnets in install-config.yaml.

Reply:
Yeah, I understood. From install workflow management view, it is better to give it an accurate name, once we have day1 install way, then we can develop a new install workflow, avoid be confused by the two install workflow.

Reply:
Okay, if that is your plan, then keep it as your desire.